gulp-tasks/wfContributors.js

```diff
@@ -59,6 +64,30 @@ function buildIndex(contributors) {
   fs.writeFileSync(DEST_LIST, result);
 }
 
+function buildIndividualPages(contributors) {
+  gutil.log(' ', 'Building individual pages of all contributors...');
+  var files = wfHelper.getFileList(GLOBAL.WF.src.content, ['**/*.md']);
+  var filesByAuthor = wfHelper.splitByAuthor(files);
+
+  var keys = Object.keys(contributors);
+  keys.forEach(function(key) {
+    if (!(key in filesByAuthor)) {
+      return;
+    }
+    filesByAuthor[key].sort(wfHelper.updatedComparator);
+    var name = contributors[key].name;
+    var ts = fs.readFileSync(TEMPLATE_ARTICLE_LIST, 'utf8');
+    var template = Handlebars.compile(ts);
+    var context = {
+      title: 'Latest contributions from ' + name.given + ' ' + name.family,
+      articles: filesByAuthor[key],
+    };
+    var result = template(context);
+    var outputFile = path.join(DEST_ARTICLE_LIST, key, 'index.md');
```

Comment:
Instead of dropping each contributor into it's own folder (`/contributors/petele/index.md`), let's put them in the contributors folder and use the key as the file name (eg `/contributors/petele.md`)

Reply:
What if someone is called `index`?

Reply:
Good call! We should add that as a check to `testContributors` in `gulp-tasks/test.js`!

Reply:
Done